presc/misclassifications/misclass_rate.py

```diff
@@ -123,8 +154,51 @@ def misclass_rate_feature(
             rate += [float("nan")]
             standard_deviation += [float("nan")]
     misclass_rate_histogram = rate
-
-    return bins, misclass_rate_histogram, standard_deviation
+    
+    
+    # compute metrics
+    
+    metric_list=[]
+    
+    if metric == "accuracy score":
+        accuracy_score_list=[]
+
+        for index in range(1,len(total_histogram_counts)+1):
```

Comment:
It would be better not to keep repeating this code block. Looks like the only thing that changes is the metric function? This can be saved to a variable above, eg. `metric_fun = accuracy_score`, and then use `sc = metric_fun(test_y, pred_y)`. In fact, if the `metric` param accepts a function as discussed above, you could just use that function directly (`metric(test_y, pred_y)`).

Reply:
It is a really good idea. I have modified it.